### DISCUSSION TITLE
RX-V4A  works as well

### DISCUSSION THREAD

**Discussion by Anybody4u** [mvdwetering/yamaha_ynca#354] [Show and tell]
I am successfully using the Yamaha YNCA integration with a Yamaha RX-V4A receiver.
At first the receiver was not recognized by the integration, but on the 3rd run the receiver was recognized via its IP address. Since then it has been working perfectly.

**Comment by mvdwetering** [comment]
Thanks for the report 👍
I added RX-V4A to the list in the readme
